docs/_articles/en/user-guide/org-browser.md

```diff
@@ -34,9 +35,14 @@ You can refresh metadata at org level, for a type, for folders in a type, and fo
 
 ![Metadata type with folders](./images/org_browser_folders.png)
 
-## Retrieving Components
+## Retrieving Metadata
 
-You can retrieve a component to your local project by clicking the retrieve button **(7)** next to the component name. Currently, the component is retrieved into your default package directory, which is defined in the [sfdx-project.json](./en/getting-started/first-project#the-sfdx-projectjson-file) file.
+You can retrieve a metadata component or all metadata of the same type. Click retrieve icon next to:
```

Comment:
```suggestion
You can retrieve a single metadata component, or all components of the same type. Click retrieve icon next to:
```

Reply:
Done

Reply:
@smaddox-sf , @brpowell , let me know if you have any more comments...